src/core/export/export.service.js

```diff
@@ -0,0 +1,22 @@
+import {get as valueFactory} from '@grid/core/services/value';
+
+export function toCsv(rows, columns) {
+	const CSV_DELIMETER = ',';
+	const columnsArr = [];
+	const rowsArr = [];
+	const head = columns.map(column => {
+		return column.title;
+	}).join(CSV_DELIMETER);
+
+	rowsArr.push(head);
+	rows.map(item => {
+		columnsArr.push(columns.map(column => {
+			const value = valueFactory(item, column).toString();
```

Comment:
this is bad idea to use valueFactory in forEach need to cache it,
don't use toString() look to stringify method in cell-value directive